src/assets/scss/style.scss

```diff
@@ -0,0 +1,3 @@
+@import '../../../node_modules/bootstrap/scss/bootstrap';
+
+@import 'general';
```

Comment:
Being picky, but can we add a line at the end of the file? That's ESLint convention, but I think it doesn't check the scss files!